systemtest/src/test/java/io/strimzi/systemtest/specific/SpecificST.java

```diff
@@ -114,21 +114,15 @@ void testLoadBalancerIpOverride() throws Exception {
     @Test
     @Tag(REGRESSION)
     void testDeployUnsupportedKafka() {
-        KafkaResource.kafkaWithoutWait(KafkaResource.defaultKafka(CLUSTER_NAME, 1, 1).build());
+        String nonExistingVersion = "6.6.6";
+        KafkaResource.kafkaWithoutWait(KafkaResource.defaultKafka(CLUSTER_NAME, 1, 1).editSpec().editKafka().withVersion(nonExistingVersion).endKafka().endSpec().build());
 
         LOGGER.info("Wait until Zookeeper stateful set is ready");
         StatefulSetUtils.waitForAllStatefulSetPodsReady(KafkaResources.zookeeperStatefulSetName(CLUSTER_NAME), 1);
 
-        KafkaUtils.waitUntilKafkaStatusConditionIsPresent(CLUSTER_NAME);
+        String coLog = kubeClient().logs(kubeClient().listPodNames("name", "strimzi-cluster-operator").get(0));
 
-        Condition condition = KafkaResource.kafkaClient().inNamespace(NAMESPACE).withName(CLUSTER_NAME).get().getStatus().getConditions().get(0);
-
-        verifyCRStatusCondition(
-                condition,
-                "Version 6.6.6 is not supported.",
-                "InvalidResourceException",
-                "True",
-                "NotReady");
+        assertThat(coLog.contains("Version " + nonExistingVersion + " is not supported"), is(true));
```

Comment:
Why did you removed the status check? It should IMHO be there. You are also not waiting for things to finish. So I wonder how do you know that it should be already in the log.